app/build.gradle

```diff
@@ -174,6 +174,7 @@ dependencies {
     svrImplementation fileTree(dir: "${project.rootDir}/third_party/svr/", include: ['*.jar'])
     implementation 'com.android.support:design:26.1.0'
     implementation 'com.google.vr:sdk-audio:1.140.0'
+    implementation "org.mozilla.components:telemetry:$mozilla_components_version"
```

Comment:
Set the version here instead of using $mozilla_components_version to be consistent with the other dependencies